packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -55,6 +55,7 @@ const AppConfig = ({children, locals = {}}) => {
             redirectURI={`${appOrigin}/callback`}
             proxy={`${appOrigin}${commerceApiConfig.proxyPath}`}
             headers={headers}
+            // clientSecret="enable-client_secret"
```

Comment:
[NIT] Would it make sense to leave a note of why this is commented out?

Reply:
Good point. I will add a comment here in a follow up PR.

Reply:
A comment was added in #1676